LocalSettings.php

```diff
@@ -3746,6 +3746,11 @@
 			"*",
 		],
 	],
+	
+	// ScratchBlocks
+	'wgScratchBlocks4BlockVersion' => [
+		'default' => '3',
```

Comment:
Remove the quotes, this just needs to be int it seems as that is what default in MWS is.

Reply:
Done.